src/libraries/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -250,6 +250,44 @@ public static void IgnoredDataShouldNotBeExtensionData()
             Assert.Null(obj.MyOverflow);
         }
 
+
+        private class ClassWithExtensionData
+        {
+            [JsonExtensionData]
```

Comment:
Let's add and test another object where the converter is added with `JsonConverterAttribute`.

Reply:
done